projects/plugins/jetpack/extensions/blocks/gif/block.json

```diff
@@ -30,12 +30,21 @@
 		"paddingTop": {
 			"type": "string",
 			"default": "56.2%"
+		},
+		"attributionUrl": {
+			"type": "string"
+		},
+		"attributionName": {
+			"type": "string"
+		},
+		"giphyUrl": {
+			"type": "string"
 		}
 	},
 	"example": {
 		"attributes": {
 			"align": "center",
-			"giphyUrl": "https://giphy.com/embed/fxKWgoOG9hzPPkE1oc",
+			"gifUrl": "https://64.media.tumblr.com/3131b9d2a05485e7ae7838a8b1314ab2/2d72fb0f707ea67a-5f/s500x750/d45e10288e4ff6a775c879978f52c87d3318ff8d.gif",
```

Comment:
Any chance you could find an image with a different aspect ratio? It doesn't look great right now:

<img width="665" alt="image" src="https://github.com/user-attachments/assets/28a41aed-65c0-4dfe-85cf-cb4749380f14">